packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -194,4 +205,19 @@ export class AndroidSdkTools {
     ];
     await util.execute(installCmd, env, this.log);
   }
+
+  /**
+   * Checks if the androidSdkPath in the config file is valid.
+   * @param {Config} config the bubblewrap general configuration.
+   */
+  static async validatePath(config: Config): Promise<Result<boolean, Error>> {
```

Comment:
We can pass just the path instead of the entire config and make the function more generic - this allows using this for validating input too:

```suggestion
  static async validatePath(path: string): Promise<Result<boolean, Error>> {
```

Reply:
I'm calling it sdkPath instead, we already import `path` so this name is taken

Reply:
SGTM